### PR TITLE
Rename app

### PR DESCRIPTION
**Pull request by KateF20**

### CODE REVIEW

File: taskmanager/taskeditor/templates/registration/login.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <form method="post">
+    <h1 class="h3 mb-3 fw-normal" style="color: royalblue">Log in</h1>
+      {% csrf_token %}
+      {{ form.as_p }}
+    <button class="w-100 btn btn-lg btn-primary" type="submit"
+            style="color: antiquewhite; background-color: royalblue">Sign in</button>
+  </form>
+    </main>
+</body>
+</html>
```
Comment (homm) [inline taskmanager/taskeditor/templates/registration/login.html]: No newline at the end

File: taskmanager/taskeditor/urls.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 from . import views
 
 urlpatterns = [
-    path('', views.index, name='main'),
+    path('', views.index, name='taskeditor'),
```
Comment (homm) [inline taskmanager/taskeditor/urls.py]: Not agree

File: taskmanager/taskeditor/views.py [hunk 3]
```diff
@@ -31,4 +31,4 @@ def create(request):
 
 
 def home(request):
-    return render(request, 'registration/home.html')
+    return render(request, 'main/home.html')
```
Comment (homm) [inline taskmanager/taskeditor/views.py]: This template is not exist

File: taskmanager/taskeditor/apps.py
Comment (homm) [inline taskmanager/taskeditor/apps.py:1]: Not sure if this file required

**Review by homm — CHANGES_REQUESTED** [review]

File: taskmanager/taskeditor/templates/registration/login.html (hunk 1 above)
Comment (KateF20) [reply taskmanager/taskeditor/templates/registration/login.html]: done

**Review by homm — APPROVED** [review]

**Pull request merged**